tests/base/test_extractors.py

```diff
@@ -238,3 +252,14 @@ def test_spacy_ner_extractor(spacy_nlp):
         'value': 'West',
         'entity': 'LOC',
         'confidence': None}
+
+    # Test dimension filtering
+
+    example = Message("anywhere in the West", {
+        "intent": "restaurant_search",
+        "entities": [],
+        "spacy_doc": spacy_nlp("anywhere in the west")})
+    ext = SpacyEntityExtractor({"dimensions": ["EVENT"]})
+    ext.process(example, spacy_nlp=spacy_nlp)
+
+    assert len(example.get("entities", [])) == 0
```

Comment:
Same comment here as in the duckling one -- would be great if we could test inclusion and exclusion within the same message.

Reply:
I added this with https://github.com/RasaHQ/rasa_nlu/pull/1718/commits/9df258e10acbae9fd81c7bddc10d69ad66fabb70

Reply:
Fixed in https://github.com/RasaHQ/rasa_nlu/pull/1718/commits/4045d8722dbbfa0e56026e8ba0bf68add4cfb240